src/resources/js/components/EditPresentation.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-text-field
+            v-model="updatePresentationInfo.title"
+            :rules="titleRules"
+            label="Titre"
+      />
+
+      <v-datetime-picker
+        label="Date et Heure"
+        v-model="updatePresentationInfo.date"
+        clearText="annuler"
+      >
+        <template slot="dateIcon">
+          <v-icon>mdi-calendar</v-icon>
+        </template>
+        <template slot="timeIcon">
+          <v-icon>mdi-clock</v-icon>
+        </template>
+      </v-datetime-picker>
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="submitChange"
+            >
+            Appliquer
+      </v-btn>
+    </v-form>
+</template>
+
+<script>
+import dateFormat from 'dateformat';
+
+  let alert = {};
+  export default {
+    props: ['parentRefs'],
+    data() {
+      return {
+        valid: true,
+        titleRules: [
+          (v) => v === '' || (v && v.length <= 20) || 'Le titre doit contenir moins de 20 caractères',
+        ],
+        updatePresentationInfo: {
+          date: '' ,
+          title: '',
+       },
+      };
+    },
+    beforeMount() {    
+      alert = this.parentRefs.alert;
+      
+      window.axios
+        .get(`presentations/${this.$route.params.idPresentation}`)
+        .then((response) =>{ 
+          this.updatePresentationInfo.title = response.data.title;
+          this.updatePresentationInfo.date = dateFormat(response.data.date, 'yyyy-mm-dd HH:MM');
+        })
+        .catch(() => {
+          alert.showMessage('error', 'Une erreur est survenue lors du traitement de votre demande');
+        })
+    },
+    methods: {
+      submitChange() {
+        const data = this.updatePresentationInfo;
+        const apiUrl = `presentations/${this.$route.params.idPresentation}`;
+
+        // sends credentials to backend
+        window.axios.put(apiUrl, {
+          title: this.updatePresentationInfo.title,
+          date: dateFormat(this.updatePresentationInfo.date, 'yyyy-mm-dd HH:MM'),
+        })
+        .then((response) => {
+          alert.showMessage('success', 'Les informations de la présentation ont été modifiées');
```

Comment:
when updating an objet the routes returns the new modifed object you should affect it to updatePresentationInfo to make sure that your object has the exact same values as in db

Reply:
Changed in 0e22a14